bring_api/__init__.py

```diff
@@ -1,5 +1,7 @@
 """Bring API package."""
 
+__VERSION__ = "0.8.1a1"
```

Comment:
i think this should be lowercase

Reply:
ohh, ok. I have it CAPS in all my projects, don't know why exactly. But you are right, can only find lower as well^^
https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package